app/models/planning_application.rb

```diff
@@ -219,11 +219,8 @@ def full_address
   end
 
   def secure_change_url(application_id, secure_token)
-    case ENV["DOMAIN"]
-    when "bops-staging.services"
-      "http://#{local_authority.subdomain}.bops-applicants-staging.services/change_requests?planning_application_id=#{application_id}&change_access_id=#{secure_token}"
-    when "bops.services"
-      "http://#{local_authority.subdomain}.bops-applicants.services/change_requests?planning_application_id=#{application_id}&change_access_id=#{secure_token}"
+    if ENV["DOMAIN"]
```

Comment:
Why do we have this `if` here?

Reply:
To differentiate between http and https - ENV["DOMAIN"] is only set on Prod and Staging. Ideally we'd replace this with a ENV["PROTOCOL"] but that would mean another Ansible update on both Prod and Staging that we would have to coordinate.

Reply:
What about `if Rails.env.production?`

Reply:
Much better, thanks! Will do now.